test/service/test_manifest_endpoints.py

```diff
@@ -51,14 +51,17 @@ def test_manifest_endpoint_start_execution(self, mock_uuid, current_request, ste
         mock_uuid.return_value = execution_name
         step_function_helper.describe_execution.return_value = {'status': 'RUNNING'}
         filters = {'file': {'organ': {'is': ['lymph node']}}}
-        current_request.query_params = {'filters': json.dumps(filters)}
+        format = 'tsv'
+        current_request.query_params = {'filters': json.dumps(filters),
+                                        'format': format}
```

Comment:
Don't pass format here to prove that the default works.

Reply:
I tried several variations. It always comes down to the missing format parameter in `start_manifest_generation` in `ManifestService`.